kube/src/config/mod.rs

```diff
@@ -12,7 +12,7 @@ mod utils;
 use crate::{error::ConfigError, Result};
 use file_loader::ConfigLoader;
 pub use file_loader::KubeConfigOptions;
-pub(crate) use utils::read_file_to_string;
+#[allow(unused_imports)] pub(crate) use utils::read_file_to_string;
```

Comment:
```suggestion
pub(crate) use utils::read_file_to_string;
```

Compiles without it. (`mod config;` is `#[cfg(feature = "client")]` and this function is used in `service`.)

Reply:
ah, yes, that was a relic of trying to have both a client and a config feature.